interpret_date.gemspec

```diff
@@ -18,8 +18,8 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ["lib"]
 
-  spec.add_runtime_dependency "activerecord", ">=4.2"
+  spec.add_runtime_dependency "activerecord", "~> 4.2"
```

Comment:
Are we sure that we want to do this? This would mean that we wouldn't be able to use this with the applications that are on Rails 5.x.

Reply:
Good point. I'll update this.